src/foundry/foundry.js/clientDocuments/actor.d.ts

```diff
@@ -25,10 +27,267 @@ declare global {
+      user: BaseUser
+    ): Promise<void>;
+
+    /**
+     * {@inheritdoc}
+     * @override
+     */
+    _onUpdate(
+      data: DeepPartial<foundry.documents.BaseActor>['data']['_source'],
+      options: DocumentModificationOptions,
+      user: string
+    ): void;
+
+    /**
+     * {@inheritdoc}
+     * @override
+     */
+    _onCreateEmbeddedDocuments(
+      embeddedName: string,
+      documents: foundry.abstract.Document<any, any>[],
+      result: Record<string, unknown>[],
+      options: DocumentModificationOptions,
+      userId: string
+    ): void;
+
+    /**
+     * {@inheritdoc}
+     * @override
+     */
+    _onUpdateEmbeddedDocuments(
+      embeddedName: string,
+      documents: foundry.abstract.Document<any, any>[],
+      result: Record<string, unknown>[],
+      options: DocumentModificationOptions,
+      userId: string
+    ): void;
+
+    /**
+     * {@inheritdoc}
+     * @override
+     */
+    _onDeleteEmbeddedDocuments(
+      embeddedName: string,
+      documents: foundry.abstract.Document<any, any>[],
+      result: string[],
+      options: DocumentModificationContext,
+      userId: string
+    ): void;
+
+    /**
+     * Refresh the display of active Tokens for this Actor if ActiveEffects were changed
+     */
+    protected _refreshTokens(): void;
+
+    /**
+     * You are referencing Actor#_data which has been deprecated in favor of Actor#data#_source. Support for this reference will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    get _data(): this['data']['_source'];
+
+    /**
+     * You are referencing Actor#getOwnedItem(itemId) which is deprecated in favor of Actor#items#get(itemId). Support will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    getOwnedItem(itemId: string): InstanceType<ConfiguredDocumentClass<typeof BaseItem>>;
+
+    /**
+     * You are referencing Actor#createOwnedItem which is deprecated in favor of Item.create or Actor#createEmbeddedDocuments. Support will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    createOwnedItem(
+      itemData: Parameters<this['createEmbeddedDocuments']>[1],
+      options: Parameters<this['createEmbeddedDocuments']>[2]
+    ): ReturnType<this['createEmbeddedDocuments']>;
+
+    /**
+     * You are referencing Actor#updateOwnedItem which is deprecated in favor of Item#update or Actor#updateEmbeddedDocuments. Support will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    updateOwnedItem(
+      itemData: Parameters<this['updateEmbeddedDocuments']>[1],
+      options: Parameters<this['updateEmbeddedDocuments']>[2]
+    ): ReturnType<this['updateEmbeddedDocuments']>;
+
+    /**
+     * You are referencing Actor#deleteOwnedItem which is deprecated in favor of Item#delete or Actor#deleteEmbeddedDocuments. Support will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    deleteOwnedItem(
+      itemData: Parameters<this['deleteEmbeddedDocuments']>[1],
```

Comment:
the parameter is called `itemId` here. Also, it may either be an array or a single value.